### PR TITLE
[deps] Update storybook and dev react

### PR DESCRIPTION
**Pull request by noratarano**
### Summary

Update dev dependencies like storybook and the dev version of react.

### Testing

Storybook | Screenshot
--- | ---
Default | Screen Shot 2019-08-15 at 6 27 38 PM
CSS | Screen Shot 2019-08-15 at 6 27 07 PM

### Reviewers

@majapw @ljharb @goatslacker @TaeKimJR @ahuth @lencioni

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,14 +32,18 @@
...
+    "@storybook/addons": "^5.1.11",
+    "@storybook/react": "^5.1.11",
     "acorn": "^6.3.0",
     "acorn-jsx": "^5.0.1",
     "airbnb-browser-shims": "^3.2.1",
     "aphrodite": "^2.3.1",
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.3",
     "babel-jest": "^21.2.0",
+    "babel-loader": "^7.1.5",
```
Comment (noratarano) [inline package.json]: Was this installed by the storybook script or by me? 🤔

File: package.json [hunk 2]
```diff
@@ -48,8 +52,10 @@
     "chai-enzyme": "^1.0.0-beta.1",
     "clean-css": "^4.2.1",
     "create-react-class": "^15.6.3",
+    "css-loader": "^3.2.0",
```
Comment (noratarano) [inline package.json]: Oops

File: package.json (hunk 1 above)
Comment (noratarano) [reply package.json]: Storybook doesn't work without it.

File: scripts/buildCSS.js [hunk 3]
```diff
@@ -25,5 +25,5 @@ if (!fs.existsSync(dir)) {
   fs.mkdirSync(dir);
 }
 
-const outputFilePath = optimizeForProduction ? `${dir}/rheostat.css` : `${dir}/rheostat.css`;
```
Comment (noratarano) [inline scripts/buildCSS.js]: These were the same? The logic is already here: https://github.com/airbnb/rheostat/pull/244/files#diff-86fcfa6a3dc885f28527e9ff85639ed9R22

File: package.json [hunk 4]
```diff
@@ -48,8 +52,10 @@
...
     "enzyme": "^3.10.0",
-    "enzyme-adapter-react-helper": "^1.3.6",
+    "enzyme-adapter-react-16": "^1.14.0",
```
Comment (ljharb) [inline package.json]: this shouldn’t be needed what with the helper

Comment (noratarano) [reply package.json]: @ljharb `enzyme-adapter-react-16` is the same as this call `enzyme-adapter-react-install 16`?

File: package.json (hunk 1 above)
Comment (alejandrogarciasalas) [reply package.json]: yeah, it's a storybook dependency https://github.com/storybookjs/storybook/blob/next/package.json#L150

File: package.json (hunk 4 above)
Comment (ljharb) [reply package.json]: yep! it installs the right one for the version, and it’s important for testing of multiple react versions that no specific adapter is explicitly in dev deps.

**Pull request merged**